composeApp/src/commonMain/kotlin/com/majotyler/hiittimer/presentation/TimerScreen.kt

```diff
@@ -53,11 +55,24 @@ fun TimerScreen(viewModel: TimerViewModel) {
         Box(
             modifier = Modifier.fillMaxSize().background(color = Color(0xFFFFFFFF)).padding(16.dp)
         ) {
-            Column(Modifier.align(Alignment.TopStart)) {
-                Workouts(list){ item -> viewModel.onDelete(item) }
+            Column(Modifier.align(Alignment.TopStart).fillMaxWidth().padding(top = 32.dp)) {
+                Row() {
+                    Workouts(list) { index -> viewModel.onClickedDelete(index) }
+                    Spacer(Modifier.weight(1f))
```

Comment:
Instead of the `Spacer` being `weight(1F)`, that modifier should be on `Workouts` :) You should remove the "hard-coded" width from `Workouts`: this stuff `.width(200.dp)`. The spacer can be made a fixed width, like `.width(4.dp)` or, you can do this really useful thing on your `Row` where you say each element (`Workouts`, `RepsButton`) has a certain space between them. You do that like this `Row(horizontalArrangement = Arrangement.spacedBy(space = 4.dp)) {`.